### DISCUSSION TITLE
Shared Login Sample

### DISCUSSION THREAD

**Discussion by roddyvet** [Finbuckle/Finbuckle.MultiTenant#553] [General]
I am looking to implement Finbuckle MultiTenant using a shared login across tenants. There is a link in the documentation to a Shared Login sample project. The problem is this sample was in .NET Core 2.1 and doesn't appear to be in github any longer.  Is there a newer sample available that shows a shared login setup?

**Comment by AndrewTriesToCode** [comment]
Hi Roddy,
sorry for he slow reply. You can still find the sample on an older branch and it's mostly still usable:
https://github.com/Finbuckle/Finbuckle.MultiTenant/tree/v6.0.0/samples/ASP.NET%20Core%202/SharedLoginSample

I should update the docs to reflect this, or I guess update the sample to .NET 6.
